### PR TITLE
User info

### PR DESCRIPTION
**Pull request by zacpez**

### CODE REVIEW

File: cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js [hunk 1]
```diff
@@ -135,11 +168,18 @@ export default function PropertyDetails2() {
             name="opportunityLink"
           />
         </Row>
-        <Row className="mb-3">
-          <h4>Business Contract</h4>
+        <Row>
+          <div className="d-flex flex-column">
+            <h4>Business Contract</h4>
```
Comment (HypnoMama) [inline cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js:173]: Should be Business Contact

File: cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js [hunk 2]
```diff
@@ -135,11 +168,18 @@ export default function PropertyDetails2() {
...
+            <span style={{ opacity: "0.7" }} className="my-1">
+              This will the contact information displayed on the public listing
```
Comment (HypnoMama) [inline cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js:175]: needs a "be"

**Review by HypnoMama — APPROVED** [review]

File: cit3.0-web/src/store/models/opportunity.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ export const OPPORTUNITY_MODEL = () => ({
   publicNote: "",
   privateNote: "",
   lastAdmin: "",
+  user: 1,
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/store/models/opportunity.js:23]: Is this a default value that is an ID?

**Review by BrendanBeachBC — APPROVED** [review]
Looks good to me, just a couple of questions/comments

**Pull request merged**